client/src/components/BoardDashboard.js

```diff
@@ -101,32 +98,25 @@ PrototypeNode.propTypes = {
   onClick: PropTypes.func
 }
 
-const BoardDashboard = ({ pageDispatchers }) => {
+const BoardDashboard = ({
+  diagramData,
+  readonly,
+  relatedObject,
+  relatedObjectType,
+  diagramHeight,
+  setDiagramHeight,
+  onUpdate,
+  setError,
+  saveDisabled
+}) => {
   // Make sure we have a navigation menu
-  useBoilerplate({
```

Comment:
This is still used as a page in `Routing.js`, so it should stay under `pages/`, and also use the boilerplate.